src/content/structured/components/select/code.mdx

```diff
@@ -478,54 +478,39 @@ export const snippetsValidation = [
     { label: "Mocha", value: "mocha" },
   ]}
   validationStatus="success" // also shown above are warning and error
-  validationText="Success message"
+  validationText="Coffee available"
   onIcChange={(event) => console.log(event.detail.value)}
 />`,
   },
 ];
 
 export const SelectExampleValidation = () => (
   <div>
-    <IcSelect
+    <IcSelect placeholder="Americano"
```

Comment:
```suggestion
    <IcSelect placeholder="Select an option..."
```
Think the placeholder should be select an option as that matches the code snippet

Reply:
Same for the two below too